README.md

```diff
@@ -1,3 +1,142 @@
+This repository contains an up-to-date fork of
+[Wasmtime](https://docs.wasmtime.dev/), a standalone WebAssembly engine. This
+fork adds support for the [WasmFX](https://wasmfx.dev/) proposal for stack
+switching. 
```

Comment:
This repository is a fork of [Wasmtime](https://docs.wasmtime.dev/) extended with support for the [WasmFX](https://wasmfx.dev/) instruction set. WasmFX adds stack switching capabilities to core Wasm, thereby enabling interleaving of computation. The design of the instruction set extension is based on [effect handlers](https://effect-handlers.org), which provide a structured facility for handling non-local control flow.